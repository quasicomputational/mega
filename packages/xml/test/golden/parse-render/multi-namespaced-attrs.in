<foo xmlns:a="A" xmlns:b="B" a:spam="spam" a:eggs="eggs" b:ham="ham" />