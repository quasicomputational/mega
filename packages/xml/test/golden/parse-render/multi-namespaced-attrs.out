<foo n1:eggs="eggs" xmlns:n1="A" n2:spam="spam" xmlns:n2="A" n3:ham="ham" xmlns:n3="B"></foo>